src/chat/chat_panel.rs

```diff
@@ -50,11 +51,67 @@ live_design! {
     }
 
     UserChatLine = <ChatLine> {
+        margin: {left: 100}
         avatar_section = {
-            <Image> {
-                source: dep("crate://self/resources/images/chat_user_icon.png"),
-                width: 20,
-                height: 20,
+            visible: false,
+        }
+        main_section = {
+            body_section = {
+                align: {x: 1.0, y: 0.5},
+                role_layout = {
+                    visible: false,
+                }
+                bubble = {
+                    draw_bg: {
+                        color: #15859A
+                    }
+                    markdown_message_container = {
+                        markdown_message = {
+                            draw_normal: {
+                                color: (#fff),
```

Comment:
I extracted colors out because each is specific to the kind of message and just overriding one of them looks like "patching"

Reply:
This makes sense 👍 

The Markdown widget is a complex one, so yeah.. I don't see a simpler way to adjust color at the moment.